src/app/utils/render.js

```diff
@@ -0,0 +1,18 @@
+const render = (parentId, ...children) => {
```

Comment:
are you sure about the id? Maybe just a querySelector or NodeElement?

Reply:
changed to querySelector